spec/jstests-spec.js

```diff
@@ -285,39 +323,211 @@ describe("Check initializing variables are passed correctly", function() {
   });
 });
 
-describe("Node search", function() {
-  const cy = cytoscape({ 
-    elements: [
-    { data: { id: "A" } },
-    { data: { id: "B" } },
-    {
-      data: {
-        id: "AB",
-        source: "A",
-        target: "B"
-      }
-    }]
+describe("Ensuring correct nodes are highlighted in mutated graph", function () {
+  let cy;
+  beforeEach(function () {
+    document.body.innerHTML = `
+    <div id="cy"></div>`;
+    cy = cytoscape({
+      container: document.getElementById("cy"),
+      elements: [{
+        group: "nodes",
+        data: {
+          id: "A"
+        }
+      },
+      {
+        group: "nodes",
+        data: {
+          id: "B"
+        }
+      },
+      {
+        group: "nodes",
+        data: {
+          id: "edgeAB",
+          source: "A",
+          target: "B"
+        }
+      },
+      ],
+    });
   });
 
-  it("should be a successful search", function() {
-    const result = searchNode(cy, "A");
+  it("highlights an added node in green", function () {
+    const mutObj = {
+      "type_": 1,
+      "startNode_": "A"
+    };
+    const mutList = [];
+    mutList.push(mutObj);
+    highlightDiff(cy, mutList);
+
+    // expect node to be green
+    expect(cy.getElementById("A").style("background-color")).toBe('rgb(0,128,0)');
```

Comment:
You could probably define constants `red`, `green` and `yellow` to be these strings, so that you can say `expect(...).toBe(green)` without having to add the additional comment.